src/wallet/rpcwallet.cpp

```diff
@@ -5014,17 +5014,20 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
             "z_sendmany \"fromaddress\" [{\"address\":... ,\"amount\":...},...] ( minconf ) ( fee ) ( privacyPolicy )\n"
             "\nSend multiple times. Amounts are decimal numbers with at most 8 digits of precision."
             "\nChange generated from one or more transparent addresses flows to a new transparent"
-            "\naddress, while change generated from a shielded address returns to itself."
-            "\nWhen sending coinbase UTXOs to a shielded address, change is not allowed."
-            "\nThe entire value of the UTXO(s) must be consumed."
-            + strprintf("\nBefore Sapling activates, the maximum number of zaddr outputs is %d due to transaction size limits.\n", Z_SENDMANY_MAX_ZADDR_OUTPUTS_BEFORE_SAPLING)
+            "\naddress, while change generated from a legacy Sapling address returns to itself."
+            "\nWhen sending from a unified address, change is returned to the internal-only address"
+            "\nfor the associated unified account."
+            "\nWhen spending coinbase UTXOs, only shielded recipients are permitted and change is not allowed;"
+            "\nthe entire value of the coinbase UTXO(s) must be consumed."
             + HelpRequiringPassphrase() + "\n"
             "\nArguments:\n"
             "1. \"fromaddress\"         (string, required) The transparent or shielded address to send the funds from.\n"
             "                           The following special strings are also accepted:\n"
             "                               - \"ANY_TADDR\": Select non-coinbase UTXOs from any transparent addresses belonging to the wallet.\n"
             "                                              Use z_shieldcoinbase to shield coinbase UTXOs from multiple transparent addresses.\n"
-            "                           If the address is a UA, transfer from the most recent value pool with sufficient funds\n"
+            "                           If a unified address is provided for this argument, the TXOs to be spent will be selected from those\n"
+            "                           associated with the account corresponding to that unified address, from value pools corresponding\n"
+            "                           to the receivers included in the UA.\n"
```

Comment:
Is that always true? I thought that TXOs from other pools would be selected if needed and if allowed by the privacy policy (i.e. at least AllowRevealedAmounts). Or did we not implement that in the end?

Reply:
The `AccountZTXOPattern` type explicitly takes into account the receiver types: https://github.com/zcash/zcash/blob/master/src/wallet/wallet.h#L798-L800

This will be able to change once we can be smarter about interpreting the privacy policy, in #5978